pyqtorch/primitive.py

```diff
@@ -5,7 +5,7 @@
 import torch
 
 from pyqtorch.abstract import AbstractOperator
-from pyqtorch.apply import _apply_gate
+from pyqtorch.apply import _apply_tensordot
```

Comment:
Why did the name change here? `apply_operations` or `apply_gate` looked good to me, honestly.

Reply:
its called _apply_operator_ now